### ISSUE TITLE
Random crash at startup 

### ISSUE THREAD

**Issue by Starkmann** [meltzow/supernovae#37]
I experience a random crash at startup without doing anything:
```

Mai 22, 2016 2:51:12 PM com.jme3.app.LegacyApplication handleError
SCHWERWIEGEND: Uncaught exception thrown in Thread[JavaFX Application Thread,5,main]
java.lang.ArrayIndexOutOfBoundsException: -1
	at java.util.ArrayList.elementData(ArrayList.java:418)
	at java.util.ArrayList.set(ArrayList.java:446)
	at com.sun.javafx.collections.ObservableListWrapper.doSet(ObservableListWrapper.java:106)
	at javafx.collections.ModifiableObservableListBase.set(ModifiableObservableListBase.java:160)
	at javafx.beans.binding.ListExpression.set(ListExpression.java:306)
	at com.brainless.alchemist.model.ECS.data.TraversableEntityData.lambda$2(TraversableEntityData.java:116)
	at com.sun.javafx.application.PlatformImpl.lambda$null$173(PlatformImpl.java:295)
	at java.security.AccessController.doPrivileged(Native Method)
	at com.sun.javafx.application.PlatformImpl.lambda$runLater$174(PlatformImpl.java:294)
	at com.sun.glass.ui.InvokeLaterDispatcher$Future.run(InvokeLaterDispatcher.java:95)
	at com.sun.glass.ui.gtk.GtkApplication._runLoop(Native Method)
	at com.sun.glass.ui.gtk.GtkApplication.lambda$null$49(GtkApplication.java:139)
	at java.lang.Thread.run(Thread.java:745)

8446 [Thread-694] ERROR supernovae.processor.logic.command.NeededRotationProc - Exception in processor : NeededRotationProc : null
java.lang.NullPointerException
	at com.simsilica.es.base.DefaultEntitySet.entityMatches(DefaultEntitySet.java:450)
	at com.simsilica.es.base.DefaultEntitySet$Transaction.resolveChanges(DefaultEntitySet.java:878)
	at com.simsilica.es.base.DefaultEntitySet.applyChanges(DefaultEntitySet.java:406)
	at com.simsilica.es.base.DefaultEntitySet.applyChanges(DefaultEntitySet.java:344)
	at com.brainless.alchemist.model.ECS.pipeline.BaseProcessor.update(BaseProcessor.java:58)
	at com.jme3.app.state.AppStateManager.update(AppStateManager.java:287)
	at com.brainless.alchemist.model.ECS.pipeline.PipelineManager.lambda$0(PipelineManager.java:43)
	at java.lang.Thread.run(Thread.java:745)
ANIMATION must be stoped here
```

**Comment by Starkmann** [comment]
No i have some kind of a race condition while starting the game:
```
INFORMATION: Audio max auxiliary sends: 4
java.lang.NullPointerException
NewState: FAILED
ERROR AT LOADING: FAILED
```

**Comment by meltzow** [comment]
please try again. its better now? 
dont forget to pull the alchemist.

**Comment by Starkmann** [comment]
Had to fix something: c59d208d1077aa5f4b160f4acf2a28846051dbd2
But now its working like charm.
